### DISCUSSION TITLE
How to get IP address?

### DISCUSSION THREAD

**Discussion by ChristopheBougere** [deeplay-io/nice-grpc#31] [Q&A] [answered]
Hi,

I'd like to get the IP address of a call on server-side.
I believe this is possible using the [`call.getPeer()` method in `grpc-js`](https://github.com/grpc/grpc-node/pull/1526/files), however I can't manage to retrieve it from `nice-grpc`.

Am I missing something?

**Comment by aikoven** [comment, marked as answer]
It is available as a `peer` field in [`CallContext`](https://github.com/deeplay-io/nice-grpc/blob/master/packages/nice-grpc-common/src/server/CallContext.ts) that is passed as a second argument to every service implementation method.

**Reply by ChristopheBougere** [reply]
Oh I totally missed that, thanks a lot 🙏
